x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -83,38 +139,68 @@ const GraphOverlayComponent = ({
     },
     [currentTimeline, dispatch, graphEventId, navigateToApp, onCloseCaseModal, timelineId, title]
   );
+  const {
+    timelineFullScreen,
+    setTimelineFullScreen,
+    globalFullScreen,
+    setGlobalFullScreen,
+  } = useFullScreen();
+  const fullScreen = useMemo(
+    () => isFullScreen({ globalFullScreen, timelineId, timelineFullScreen }),
+    [globalFullScreen, timelineId, timelineFullScreen]
+  );
+  const toggleFullScreen = useCallback(() => {
+    if (timelineId === TimelineId.active) {
+      setTimelineFullScreen(!timelineFullScreen);
+    } else {
+      setGlobalFullScreen(!globalFullScreen);
+    }
+  }, [
+    timelineId,
+    setTimelineFullScreen,
+    timelineFullScreen,
+    setGlobalFullScreen,
+    globalFullScreen,
+  ]);
 
   return (
     <OverlayContainer bodyHeight={bodyHeight}>
       <EuiHorizontalRule margin="none" />
       <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween">
         <EuiFlexItem grow={false}>
-          <EuiButtonEmpty onClick={onCloseOverlay} size="xs">
-            {i18n.BACK_TO_EVENTS}
-          </EuiButtonEmpty>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiFlexGroup gutterSize="none">
-            <EuiFlexItem grow={false}>
-              <NewCase
-                compact={true}
-                graphEventId={graphEventId}
-                onClosePopover={noop}
-                timelineId={timelineId}
-                timelineTitle={title}
-                timelineStatus={status}
-              />
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <ExistingCase
-                compact={true}
-                onClosePopover={noop}
-                onOpenCaseModal={onOpenCaseModal}
-                timelineStatus={status}
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
+          <Navigation
+            fullScreen={fullScreen}
+            globalFullScreen={globalFullScreen}
+            onCloseOverlay={onCloseOverlay}
+            timelineId={timelineId}
+            timelineFullScreen={timelineFullScreen}
+            toggleFullScreen={toggleFullScreen}
+          />
         </EuiFlexItem>
+        {timelineId === TimelineId.active && (
```

Comment:
ok cool, I manage template timeline in my PR but not that

Reply:
😎